cartridges/int_algolia/cartridge/scripts/algolia/model/algoliaLocalizedProduct.js

```diff
@@ -467,6 +467,10 @@ var aggregatedValueHandlers = {
 
             let inStock = productFilter.isInStock(variant, ALGOLIA_IN_STOCK_THRESHOLD);
 
+            if (!productFilter.isInclude(variant)) {
+                continue;
+            }
+
             if (!inStock && !INDEX_OUT_OF_STOCK) {
```

Comment:
You have split two lines of code that were going together:
```
let inStock =
if (!inStock ...
```
Can you put this new check above or below, but not in the middle?